### PR TITLE
Added text boxes to edit the initial conditions in the GUI.

### LINKED ISSUE #206: The GUI rerun should support changing the initial conditions and time array.
There is no way to change the initial conditions or time array from the GUI. These are important items to be able manipulate.

### PR DESCRIPTION
**Pull request by moorepants**
Partially fixes #206.

- [x] There are no merge conflicts.
- [x] If there is a related issue, a reference to that issue is in the
  commit message.
- [ ] Unit tests have been added for the new feature.
- [x] The PR passes tests both locally (run `nosetests`) and on Travis CI.
- [x] All public methods and classes have docstrings. (We use the [numpydoc
  format](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt).)
- [ ] An explanation has been added to the online documentation. (`docs`
  directory)
- [x] The code follows PEP8 guidelines. (use a linter, e.g.
  [pylint](http://www.pylint.org), to check your code)
- [ ] The new feature is documented in the [Release
  Notes](https://github.com/pydy/pydy#release-notes).
- [x] The code is backwards compatible. (All public methods/classes must
  follow deprecation cycles.)
- [ ] All reviewer comments have been addressed.

### CODE REVIEW

**Comment by moorepants** [comment]
selection_050

@pydy/pydy-developers Review needed.

**Comment by oliverlee** [comment]
Can you add a short description of new feature to the release notes?

**Comment by moorepants** [comment]
Note is added. I don't think we need to update the online docs yet. There will be some more gui changes.

**Comment by moorepants** [comment]
I just tested this on Ubuntu 15.10 and it works fine.

What is this solving though? I'd hate to introduce a dependency for little reason. But this seems ok as it can be install via conda and pip.

You need to update the setup.py dependencies to have pywin32 for Windows. We'll also need to include this for Windows in the conda recipes with a selector.

**Comment by oliverlee** [comment]
There's no way to properly send signals in Windows without the package. I
think ipython and some other common packages also depend on it.

**Comment by mvnnn** [comment]
Need to space between two column.
screenshot from 2016-03-08 02 08 57

**Comment by moorepants** [comment]
It looks like you are using chrome on linux. Can you report your browser and version?

The screenshot I have above works for Firefox 44 on ubuntu.

Thanks for trying this out.

**Comment by mvnnn** [comment]
you are right. my browser is chrome and OS is Linux mint.

**Comment by oliverlee** [comment]
Same issue with me.
OSX 10.11.3 and Chrome 49.0.2623.75 (64-bit).

**Comment by oliverlee** [comment]
You should also change the initial condition labels so they aren't displayed as functions of time.

**Comment by moorepants** [comment]
I've reported the overlap issue upstream: https://github.com/ipython/ipywidgets/issues/412. Let's see what they have to say.

**Comment by moorepants** [comment]
I've improved the display of the input boxes. My solution is hacky but it seems to work ok.

selection_054

**Comment by moorepants** [comment]
Chromium has some odd vertical lines beside the mathjax.

selection_055

**Comment by oliverlee** [comment]
Looks okay in Chrome for OSX:
screen shot 2016-03-09 at 16 54 08

**Comment by moorepants** [comment]
Ok. I'm happy with this if you all are. I don't plan to add any more changes unless you all request. Please review and merge at will.

**Comment by oliverlee** [comment]
+1 Merge when CI is complete.
